app/views/layouts/global/_head.html.erb

```diff
@@ -24,4 +24,14 @@
 <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700" rel="stylesheet">
 
 <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-<%= javascript_pack_tag 'application' %>
\ No newline at end of file
+<%= javascript_pack_tag 'application' %>
```

Comment:
Is this linked to the informatics account? Might be worth only including when rails env is prod, otherwise it will include staging and probably also when you launch it locally.

Reply:
Also seems to be two properties (on which is crossed out - not sure if that means it's deleted...?) and then in the non-crossed out property there are 2 all web site data views - can you delete one of these views?

Reply:
I think (but may be wrong) that it doesn't actually care about the domain you enter (because for example when we put in the encore url it has several different ones but I think all register... I'll have a quick google!